app/controllers/report_controller/reports/editor.rb

```diff
@@ -651,12 +651,6 @@ def gfv_chargeback
       @edit[:new][:cb_tenant_id] = params[:cb_tenant_id].blank? ? nil : params[:cb_tenant_id].to_i
     elsif params.key?(:cb_tag_value)
       @edit[:new][:cb_tag_value] = params[:cb_tag_value].blank? ? nil : params[:cb_tag_value]
-    elsif params.key?(:cb_model)
```

Comment:
Seems like a leftover from when I thought we could go with a single Chargeback class.
I think this can be removed